tool_commands/command_model.py

```diff
@@ -1,62 +1,27 @@
-from functools import reduce
 from qgis.PyQt.QtGui import QIcon
 from qgis.PyQt.QtGui import QStandardItem
 from qgis.PyQt.QtGui import QStandardItemModel
-
-import os
-
-
-DEFAULT_COMMAND_DIR = os.path.join(
-    os.path.dirname(os.path.abspath(__file__)), "commands"
-)
+from ThreeDiToolbox.tool_commands.constants import step_modulename_mapping
 
 
 class CommandModel(QStandardItemModel):
-    def __init__(self, toolbox_dir=None, parent=None):
+    def __init__(self, parent=None):
         super().__init__(parent)
+        self.add_items(self)
```

Comment:
Klopt deze regel? self doorgeven? Moet dat niet self.parent zijn? Even voor de zekerheid.

Reply:
dat kan volgens mij niet..

self = <ThreeDiToolbox.tool_commands.command_model.CommandModel object at 0x7f1e9ceb53a8>
self.parent = <built-in method parent of CommandModel object at 0x7f1e9ceb53a8>
parent = None